packages/flutter_tools/lib/src/run_cold.dart

```diff
@@ -131,6 +131,13 @@ class ColdRunner extends ResidentRunner {
       await connectToServiceProtocol();
     } catch (error) {
       printError('Error connecting to the service protocol: $error');
+      // https://github.com/flutter/flutter/issues/33050
+      // TODO(blasten): Remove this check once https://b.corp.google.com/issues/132325318 has been fixed.
```

Comment:
A b/ link is okay to note here, but we should avoid full internal urls.

Reply:
This particular bug is visible to the public, so it should be okay

Reply:
Let's chat in person

Reply:
The external link is http://issuetracker.google.com/issues/132325318

Reply:
The link we should use is issuetracker.google.com/issues/132325318

Reply:
+1. Thanks!